src/OrchardCore.Modules/OrchardCore.Users/Models/RegistrationSettings.cs

```diff
@@ -2,8 +2,19 @@ namespace OrchardCore.Users.Models
 {
     public class RegistrationSettings
     {
-        public bool UsersCanRegister { get; set; }
+        public enum UsersCanRegisterEnum
```

Comment:
Extract to a new file

Can we remove the Enum suffix and use something like `RegistrationType` or something similar